custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -699,6 +701,8 @@ def _launched_col(col):
             ('num_launched_awcs', lambda col: _launched_col(col)),
             ('num_awcs_conducted_vhnd',),
             ('num_awcs_conducted_cbe',),
+            ('cbe_conducted', 'NULL'),
```

Comment:
Does this mean that this column will be NULL for all of the rows above awc level?

Reply:
Yes. Because right now it only required at awc level and levels above that its not making sense right now.